doc/WebSite/Zero/Tenant-Management.md

```diff
@@ -46,13 +46,14 @@ The AbpTenant class defines some base properties, the most important are:
 -   **TenancyName**: This is the **unique** name of a tenant in the
     application. It should not normally be changed. It can be used to
     allocate subdomains to tenants like '**mytenant**.mydomain.com'.
+    As such, it cannot contain spaces.
     Tenant.**TenancyNameRegex** constant defines the naming rule.
 -   **Name**: An arbitrary, human-readable, long name of the tenant.
--   **IsActive**: True if this tenant can use the application. If it's
-    false, no user of this tenant can login to to system.
+-   **IsActive**: True, if this tenant can use the application. If it's
```

Comment:
The comma separates the actual return value from the condition. See https://github.com/aspnetboilerplate/aspnetboilerplate/pull/2990#discussion_r164744531.